Tools/plugins/MuonPogTreeProducer.cc

```diff
@@ -369,6 +369,7 @@ void MuonPogTreeProducer::analyze (const edm::Event & ev, const edm::EventSetup
     {
       nGoodMuons = fillMuons(muons,vertexes,beamSpot);
     }
+  eventId_.nMuons = nGoodMuons;
```

Comment:
@calderona yes sorry I was meaning storing, for each muon, max-pT from the possible fits of each muon (tracker, global, picky ...).